src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -6,9 +6,9 @@
 		<PackageId>Monitor.Tests</PackageId>
 		<VersionPrefix>1.0.0-preview</VersionPrefix>
 	</PropertyGroup>
-	<!--<PropertyGroup>
-		<TargetFrameworks>netcoreapp1.1</TargetFrameworks>
-	</PropertyGroup>-->
+	<PropertyGroup>
```

Comment:
Please do not un-comment these lines. We have moved to netcoreapp2.0 and will fix these for all csprojs at a later time